.chloggen/retry_upgrades.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
```

Comment:
It seems this changes the upgrade only for the collector. I would change the component to `collector`

Reply:
done